docs/en/04_Changelogs/4.10.0.md

```diff
@@ -6,6 +6,14 @@
 
 ### Other new features
 
+#### New `SS_TEMP_DIR` environment variable
+
+This release adds support for a new `SS_TEMP_DIR` environment variable, which allows developers to control where a series of “temporary” files are created. These include [Manifests](/developer_guides/execution_pipeline/manifests), [Object Caching](/developer_guides/performance/caching) and [Partial Template Caching](/developer_guides/templates/partial_template_caching).
```

Comment:
Should mention why we're doing this i.e. the TEMP_PATH/TEMP_FOLDER constants when set as env variables break in xyz scenarios